### PR TITLE
Remove unused variables

### LINKED ISSUE #3610: Unused variables
## Your Environment

* PCL Version: HEAD

## Context
Variable are unneeded. Code can be simplified by removing them.

## Link for details
[Codacy of fork](https://app.codacy.com/manual/kunaltyagi/pcl/issues?bid=16056128&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W119LHsiaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6W119LHsiaWQiOiJMZXZlbCIsInZhbHVlcyI6WyJJbmZvIl19LHsiaWQiOiJQYXR0ZXJuIiwidmFsdWVzIjpbIjIwOTMiXX0seyJpZCI6IkF1dGhvciIsInZhbHVlcyI6W119XQ==)

### PR DESCRIPTION
**Pull request by ChakshuGupta13**
Fix #3610

### CODE REVIEW

File: tools/virtual_scanner.cpp [hunk 1]
```diff
@@ -174,7 +174,6 @@ main (int argc, char** argv)
 
   int subdiv_level = 1;
   double scan_dist = 3;
-  std::string fname, base;
```
Comment (kunaltyagi) [inline tools/virtual_scanner.cpp]: fname is used somewhere else :)

Comment (ChakshuGupta13) [reply tools/virtual_scanner.cpp]: Thanks @kunaltyagi for pointing out. Will fix it right away.

**Comment by ChakshuGupta13** [comment]
@kunaltyagi @taketwo 
After removing some earlier commits and implementing the requested changes in 6fbc12d, I force pushed. Kindly remove the requested changes for merging.

File: surface/include/pcl/surface/impl/convex_hull.hpp [hunk 2]
```diff
@@ -395,7 +395,6 @@ pcl::ConvexHull<PointInT>::performReconstruction3D (
     {
       polygons[dd].vertices.resize (3);
 
-      // Needed by FOREACHvertex_i_
```
Comment (kunaltyagi) [inline surface/include/pcl/surface/impl/convex_hull.hpp]: I think you accidentally removed the comment

Comment (ChakshuGupta13) [reply surface/include/pcl/surface/impl/convex_hull.hpp]: @kunaltyagi 
Done!

**Pull request merged**